superset-frontend/src/views/CRUD/welcome/ActivityTable.tsx

```diff
@@ -246,7 +252,8 @@ export default function ActivityTable({
         );
       },
     );
-  if (loadingState && !editedObjs) {
+
+  if ((loadingState && !editedObjs) || loadedCount < 3) {
```

Comment:
These magic numbers are quite confusing. I'm assuming `loadedCount>=3` means done fetching? Can't we just pass that in as a `doneFetching` flag?

Reply:
Also, while we're in cleanup mode, is the eslint disable still needed on line 263?

Reply:
done!